standardPage.html

```diff
@@ -31,7 +31,7 @@
         <svg id="svg_banner" viewBox="0 0 100 100"width=100% height=100%>
```

Comment:
Just being nit-picky but can you change `id="svg_banner"` to `class="standard-page-banner"`?